src/croniter/croniter.py

```diff
@@ -100,9 +120,30 @@ def is_32bit():
 DOW_FIELD = 4
 SECOND_FIELD = 5
 YEAR_FIELD = 6
-UNIX_FIELDS = (MINUTE_FIELD, HOUR_FIELD, DAY_FIELD, MONTH_FIELD, DOW_FIELD)
-SECOND_FIELDS = (MINUTE_FIELD, HOUR_FIELD, DAY_FIELD, MONTH_FIELD, DOW_FIELD, SECOND_FIELD)
-YEAR_FIELDS = (MINUTE_FIELD, HOUR_FIELD, DAY_FIELD, MONTH_FIELD, DOW_FIELD, SECOND_FIELD, YEAR_FIELD)
+UNIX_FIELDS = (
+    MINUTE_FIELD,
+    HOUR_FIELD,
+    DAY_FIELD,
+    MONTH_FIELD,
+    DOW_FIELD,
+)
+SECOND_FIELDS = (
+    MINUTE_FIELD,
+    HOUR_FIELD,
+    DAY_FIELD,
+    MONTH_FIELD,
+    DOW_FIELD,
+    SECOND_FIELD,
+)
+YEAR_FIELDS = (
+    MINUTE_FIELD,
+    HOUR_FIELD,
+    DAY_FIELD,
+    MONTH_FIELD,
+    DOW_FIELD,
+    SECOND_FIELD,
+    YEAR_FIELD,
+)
```

Comment:
i mean up to that part of the code, below, changes are mergeable without any comment.

Reply:
I think if we remove the [magic trailing commma](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma) the items that are not longer than 80 characters will not break into multi lines.

Let me try that.

Reply:
let me pull these out of this PR real quick though so we can merge the rest.